<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font Sampler</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="statsGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>641</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Stats</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTabWidget" name="statsTabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="statsTab_1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>&lt;</string>
        </attribute>
        <widget class="QTableWidget" name="statusTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>291</width>
           <height>141</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hits</string>
          </property>
         </column>
        </widget>
        <widget class="QTableWidget" name="methodTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>10</y>
           <width>291</width>
           <height>141</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Method</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hits</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Success rate</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total bytes</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QWidget" name="statsTab_0">
        <attribute name="title">
         <string>#</string>
        </attribute>
        <widget class="QTableWidget" name="dateTimeTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>291</width>
           <height>141</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Date/Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hits</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Success rate</string>
          </property>
         </column>
        </widget>
        <widget class="QTableWidget" name="thresholdTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>10</y>
           <width>291</width>
           <height>141</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Date/Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Threshold</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QWidget" name="statsTab_0">
        <attribute name="title">
         <string>&gt;</string>
        </attribute>
        <widget class="QTableWidget" name="urlTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>291</width>
           <height>141</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>URL</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hits</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Success rate</string>
          </property>
         </column>
        </widget>
        <widget class="QTableWidget" name="protocolTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>10</y>
           <width>291</width>
           <height>141</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Protocol</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hits</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Success rate</string>
          </property>
         </column>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="alertsGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>427</y>
      <width>641</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Alerts</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QListWidget" name="alertslistWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="summaryGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>641</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Monitor summary</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="logtimeLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="topURLLabel">
       <property name="text">
        <string>Most hit URL:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="totalHitsLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="avgRequestSizeLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="topStatusLabel">
       <property name="text">
        <string>Most returned status:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="sucessfulHitsLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="avgRequestSizeLabel">
       <property name="text">
        <string>Average request size:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sucessfulHitsLabel">
       <property name="text">
        <string>Total successful hits: </string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="topProtocolLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="thresholdLabeLabel">
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Treshold</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="topURLLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="topProtocolLabel">
       <property name="text">
        <string>Most used protocol</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="topStatusLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="logtimeLabel">
       <property name="text">
        <string>Log status at:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="totalHitsLabel">
       <property name="text">
        <string>Total hits:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="thresholdLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="totalBytesLabel">
       <property name="text">
        <string>Total Trasmitted bytes:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="totalBytesLabelVal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="refreshstatsLabel">
       <property name="text">
        <string>Refreshing log in :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="refreshLcdNumber">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="digitCount">
        <number>5</number>
       </property>
       <property name="intValue" stdset="0">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="flushstatsLabel">
       <property name="text">
        <string>Refreshing monitor in :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="flushLcdNumber">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>120</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_config">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="actionHelp"/>
    <addaction name="quitAction"/>
   </widget>
   <addaction name="menu_config"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settingsAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="iconText">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="markAction">
   <property name="text">
    <string>&amp;Mark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="unmarkAction">
   <property name="text">
    <string>&amp;Unmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="action_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Import configuration...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionExport_Stats">
   <property name="text">
    <string>Export Stats</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
